demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py

```diff
@@ -133,3 +136,29 @@ def is_file_structure_list(self) -> bool:
         """
         data = get_json(str(self.path))
         return isinstance(data, list)
+
+    def _unify(
+        self, dest_dir: Optional[Union[Path, str]] = None, output: str = ""
+    ) -> List[Path]:
+        """Unify JSONBasedContentObject in destination dir.
+
+        Args:
+            dest_dir: Destination directory, if not provided the destination directory will be the current working dir.
+            output: output suffix to add the destination directory.
+
+        Returns:
+            List[Path]: List of new created unified json files.
+        """
+        if dest_dir is None:
+            dest_dir = ""
+
+        # Unify step
+        return [
+            Path(
+                str(
+                    PrepareUploadManager.prepare_for_upload(
+                        input=self.path, output=Path(dest_dir, output)  # type: ignore[arg-type]
```

Comment:
is the `type: ignore[...]` necessary if we do `Path`? 
Also, please manually check to make sure it works with both a string and `Path`

Reply:
see my comment here: https://github.com/demisto/demisto-sdk/pull/2679#discussion_r1067840722

Reply:
its necessary even when we do it with path, however both string and path work

Reply:
checked it manually and its ok